src/Sulu/Bundle/ContentBundle/Resources/config/serializer/Document.BasePageDocument.xml

```diff
@@ -37,7 +37,7 @@
         <property name="structureType" type="string" groups="preview"/>
         <property name="originalLocale" type="string" groups="preview"/>
         <property name="locale" type="string" groups="preview"/>
-        <property name="path" type="string"/>
+        <property name="path" type="string" exclude="true"/>
```

Comment:
@wachterjohannes Do you maybe need that for the preview? It's making problems, because a subscriber tries to add another path later.